web/src/features/charts/graphUtils.ts

```diff
@@ -4,9 +4,10 @@ import { bisectLeft } from 'd3-array';
 // // https://observablehq.com/@d3/d3-selection-2-0
 import { scaleTime } from 'd3-scale';
 import { pointer } from 'd3-selection';
-import { TFunction } from 'i18next';
+import { t, TFunction } from 'i18next';
```

Comment:
t here should be imported and created bu the useTranslation hook I believe. I'm not sure it would update properly otherwise when the user changes language.

Reply:
good catch!